agg-ast/schema_derivation/src/schema_derivation.rs

```diff
@@ -349,6 +366,9 @@ impl DeriveSchema for Stage {
             Ok(Schema::simplify(&state.result_set_schema).to_owned())
         }
 
+        /// bucket_output_derive_keys is a helper for the $bucket and $bucketAuto stages.
+        /// It takes produces either a schema based on the specified output, with "count" as the
```

Comment:
I'm not sure this is grammatically correct.

Reply:
jeez this is so bad hahaha